src/components/_account-tab.scss

```
@@ -53,8 +53,8 @@
 .controller-account-profiles .ncgr-account-tab__list-item--profiles,
 .controller-account-bank_accounts .ncgr-account-tab__list-item--bank-accounts,
 .controller-account-mail_receive_settings .ncgr-account-tab__list-item--mail-receive-settings,
-.controller-account-authorized_apps .ncgr-account-tab__list-item--authorized-apps,
-.controller-account-coupons .ncgr-account-tab__list-item--coupons,
+.controller-account-authorized_apps　.ncgr-account-tab__list-item--authorized-apps,
```

@kyuurix3 

全角が入ってそう！

```suggestion
.controller-account-authorized_apps .ncgr-account-tab__list-item--authorized-apps,
```

---

がああああ！！修正します🙇‍♀️